From 01e97f0b9df9e6a1ad6a0a0320f7087c7b2a94e8 Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Sat, 26 Nov 2016 05:23:46 -0200
Subject: [PATCH] Fix utox target installation rule

CMake install rules for files generated by the build process must specify
they reference a target to be correctly looked up in the build directory,
instead of the source directory.

Signed-off-by: Ismael Luceno <ismael@iodev.co.uk>
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 98824775c76d..380fa684f219 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,9 +144,6 @@ elseif(UNIX) # xlib by default
     target_link_libraries(utoxNATIVE icon v4lconvert X11 Xext Xrender fontconfig freetype resolv dl )
 
 	install(FILES
-	  utox
-	  DESTINATION "bin")
-	install(FILES
 	  src/utox.desktop
 	  DESTINATION "share/applications")
 	install(FILES
@@ -270,3 +267,6 @@ target_link_libraries(utox
                       sodium        pthread     m   vpx )
 
 set_property(TARGET utox PROPERTY C_STANDARD 99)
+if(UNIX)
+	install(TARGETS utox RUNTIME DESTINATION "bin")
+endif()
-- 
2.10.2

